CHANGES.rst

```diff
@@ -635,6 +635,7 @@ Improvements
 - Images can now be uploaded by pasting or dropping them into the editor for minutes
   or the event description (:pr:`5458`)
 - Add JSON export for contribution details (:pr:`5460`)
+- Show which persons are external in the user search dialog (:pr:`6074`)
```

Comment:
move this to v3.3 - even more so if this is a jacow request who are already on 3.3 anyway

Reply:
oops thats a typo - people were looking at me 😄